packages/node_modules/@webex/internal-plugin-calendar/src/index.js

```diff
@@ -82,6 +103,44 @@ registerInternalPlugin('calendar', Calendar, {
             ctx.transform('decryptTextProp', 'spaceURL', object.encryptionKeyUrl, object)
           ].concat(decryptedParticipants));
         }
+      },
+      {
+        name: 'transformMeetingNotes',
+        direction: 'inbound',
+        fn(ctx, object) {
+          if (!object) {
+            return Promise.resolve();
+          }
+
+          if (!object.encryptionKeyUrl) {
+            return Promise.resolve();
+          }
+
+          return Promise.all([
+            ctx.transform('decryptTextProp', 'encryptedNotes', object.encryptionKeyUrl, object)
+          ]);
+        }
+      },
+      {
+        name: 'transformMeetingParticipants',
+        direction: 'inbound',
+        fn(ctx, object) {
+          if (!object) {
+            return Promise.resolve();
+          }
+
+          if (!object.encryptionKeyUrl) {
+            return Promise.resolve();
+          }
+
+          // Decrypt participant properties
+          const decryptedParticipants = object.encryptedParticipants.map((participant) => Promise.all([
```

Comment:
This seems to be the same transform that is in `transformMeeting`, why the duplication?

Reply:
Michael suggested to write new transforms for decrypting meeting notes and participants that are fetched separately, instead of fetching in the existing transform where it would decrypt via the current process.

Reply:
@prsaroha said that the participants can come from the meeting request if they are less than a certain number. This transform would be for the participants request if it didn't show up in the original meeting request.